draft-vvv-tls-alps.md

```diff
@@ -163,8 +163,19 @@ exchange is performed in three steps:
 {: #alps-full title="ALPS exchange in a full TLS handshake"}
 
 A TLS client can enable ALPS by specifying an `application_settings` extension
-in the ClientHello message. The value of the `extension_data` field for this
-extension SHALL be a ApplicationSettingsSupport struct:
+in the ClientHello message.
+
+~~~
+   enum {
+      application_settings(0x44cd), (65535)
+   } ExtensionType;
+~~~
+
+The `application_settings` ExtensionType value 17613 (or 0x44cd) is random
+selected for early experimental deployments. The following code points:
+17513 (or 0x4469) and 17613 (or 0x44cd) have been using for early experimental
+deployments. The value of the `extension_data` field for this extension SHALL be
+a ApplicationSettingsSupport struct:
```

Comment:
First, "is random selected" should be "was randomly selected", but also how the value was chosen doesn't belong here.

More importantly, since you all haven't really started engaging with the standards process yet, it is far too early to declare this codepoint to be the value of `application_settings`. Instead, you should revert all the changes here and simply add a sentence:

> An early experimental deployment of this protocol used the value 17513 for `application_settings`. It was then replaced with an experimental deployment which used the value 17613.

Reply:
sure, update the descriptions for this part